.expeditor/scripts/finish_release/sync_acceptance.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -eou pipefail
+
+tempdir="$(mktemp --directory --tmpdir="$(pwd)" -t "downloads-XXXX")"
+
+trap 'rm -f $tempdir' INT TERM EXIT
+
+hab_idents=(core/hab
```

Comment:
We could use a toml file here to describe these, example here: https://github.com/habitat-sh/builder/blob/master/.expeditor/builder_seed.toml

and this could then be reduced to:
```
hab pkg download --file <toml filename> --download-directory "${tempdir}"
hab pkg bulkupload \
        --url "https://bldr.acceptance.habitat.sh" \
        --auth "${PIPELINE_ACCETPANCE_AUTH_TOKEN} \
        --channel 'stable' \
        "${tempdir}"
```

Reply:
Hearty agreement! :+1:

Reply:
yes. waaay better.

Reply:
changes made @smacfarlane and @christophermaier